app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,83 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    @tweet = Tweet.all
+    @user = User.current_user(session)
+    @user_all = User.all
+    if User.is_logged_in?(session)
+      erb :'/tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if User.is_logged_in?(session)
+      @user = User.current_user(session)
+      erb :'/tweets/create_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new", locals: {message: "Your tweet was empty."}
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(:content => params[:content], :user_id => @user.id)
+      redirect to "/tweets/#{@tweet.id}", locals: {message: "Successfully created tweet."}
+    end
+  end
+
+  get '/tweets/:id' do
+    if User.is_logged_in?(session)
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'tweets/show_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    @tweet = Tweet.find_by(params[:id])
+    if User.is_logged_in?(session)
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (not just that any user is logged in) before allowing them to edit?